.github/workflows/latest_dependency_checker.yml

```
@@ -25,9 +25,9 @@ jobs:
           commit-message: Update latest dependencies
           title: Automated Latest Dependency Updates
           author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
-          body: "This is an auto-generated PR with **latest** dependency updates.
-                 Please do not delete the `latest-dep-update` branch because it's needed by the auto-dependency bot."
+          body: "This is an auto-generated PR with **latest** dependency updates."
           branch: latest-dep-update
           branch-suffix: short-commit-hash
+          delete-branch: true
           base: main
```

can we add assignees?
```suggestion
          base: main
          assignees: machineFL
```